kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -2170,4 +2172,19 @@ public String getCamsComplexMaintenanceDocumentLookup() {
         return managedLists;
     }
 
+    public List<Note> getBoNotes() {
+        if (!StringUtils.isEmpty(getObjectId())) {
+            boNotes = SpringContext.getBean(NoteService.class).getByRemoteObjectId(getObjectId());

```

Comment:
Ok thanks Mike. I just learned services are the only "injectable" objects due to the way Kuali implemented some things.